public/apps/configuration/panels/tenant-list/configure_tab1.tsx

```diff
@@ -14,66 +14,38 @@
  */
 
 import {
-  EuiBadge,
```

Comment:
Cleanup unused imports from file I took inspiration from

Reply:
@derek-ho Is there any lint rule we can add to ensure that unused imports are removed with `yarn lint:es --fix`?